src/surforama/io/mesh.py

```diff
@@ -23,3 +26,53 @@ def read_obj_file(file_path):
     values = np.ones((len(verts),))
 
     return verts, faces, values
+
+
+def convert_mask_to_mesh(
+    mask: np.ndarray,
+    barycentric_area: float = 1.0,
+    smoothing: int = 10,
+):
+    """
+    Convert a binary mask to a mesh.
+
+    Parameters
+    ----------
+    mask : np.ndarray
+        A binary mask.
+    barycentric_area : float, optional
+        The area of each triangle in the mesh, by default 1.0
+    smoothing : int, optional
+        Number of iterations for mesh smoothing, by default 10
```

Comment:
I think it would be helpful to define which smoothing algorithm is used.

Reply:
Good point. It's Laplacian smoothing. I updated it in the docstring